### PR TITLE
acceptance tests for exclusive locks

### LINKED ISSUE #34357: api acceptance tests for exclusive locks
there can be only one exclusive lock on a ressource

### PR DESCRIPTION
**Pull request by individual-it**
## Description
test to check if exclusive locks are enforced correctly

## Related Issue

- Fixes #34357

## Motivation and Context
more tests

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    And 1 locks should be reported for file "PARENT/CHILD/child.txt" of user "user0" by the WebDAV API
+    Examples:
+      | dav-path | lock-scope |
+      | old      | shared     |
+      | old      | exclusive  |
+      | new      | shared     |
+      | new      | exclusive  |
+
+  @issue-34358
+  Scenario Outline: if a child ressource is exclusivly locked a parrent ressource cannot be locked again
```
Comment (dpakach) [inline tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature]:
```suggestion
  Scenario Outline: if a child ressource is exclusivly locked a parent resource cannot be locked again
```

File: tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    #Then the HTTP status code should be "423"
+    #And 1 locks should be reported for file "PARENT/CHILD/child.txt" of user "user0" by the WebDAV API
+    Examples:
+      | dav-path | lock-scope |
+      | old      | shared     |
+      | old      | exclusive  |
+      | new      | shared     |
+      | new      | exclusive  |
+
+  Scenario Outline: if a child ressource is exclusivly locked a parrent ressource can be locked with depth 0
```
Comment (dpakach) [inline tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature]:
```suggestion
  Scenario Outline: if a child ressource is exclusivly locked a parent resource can be locked with depth 0
```

**Comment by individual-it** [comment]
@dpakach fixed typos

**Review by davitol — APPROVED** [review]
LGTM

**Comment by individual-it** [comment]
backport in  #34363

**Pull request merged**